### ISSUE TITLE
Double-Click Speed Not Adhearing to Mouse Properties

### ISSUE THREAD

**Issue by AdamRoden** [microsoft/vscode#175345]
Type: <b>Bug</b>

I have low dexterity and cannot perform a fast double-click, so in my "Mouse Properties" for Windows I have the "Double-click speed" set to the slowest option available . In every other text-box on my computer, including this submission box, I can double-click to select a word. However, in the VS Code Text Editor control I am only fast enough 1% of the time. 

VS Code version: Code 1.75.1 (441438abd1ac652551dbe4d408dfcec8a499b8bf, 2023-02-08T21:32:34.589Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 3696)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.94GB (9.66GB free)|
|Process Argv|--crash-reporter-id 96798106-d6b8-4976-a2b9-47d9ab353f14|
|Screen Reader|no|
|VM|67%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
csharp|ms-|1.25.4
isort|ms-|2022.8.0
python|ms-|2023.2.0
vscode-pylance|ms-|2023.2.40
jupyter|ms-|2023.1.2010391206
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.14
vscode-jupyter-cell-tags|ms-|0.1.6
vscode-jupyter-slideshow|ms-|0.1.5

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242:30382549
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vscaat:30438848
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
pynewext54:30669237
azure-dev_surveyone:30548225
vsccc:30610678
pyindex848:30662994
nodejswelcome1cf:30587006
3biah626:30602489
pyind779:30671433
f6dab269:30613381
pythonsymbol12:30671437
a9j8j154:30646983

```

</details>

**Comment by isidorn** [comment]
@AdamRoden thanks for opening this request. In order to implement this we would need some way to figure out the Mouse Properties from Windows. That is probably not possible from Electron @deepak1556 
Alternative approach would be to introduce a VS Code settings that users could manually set.

**Comment by AdamRoden** [comment]
@isidorn Given the limitation, I feel the prudent solution would be to set the VS Code double-click listener timeout to the slowest speed a double-click can be configured in Windows. Right now it is too short to even detect double-clicks from my mouse input simulator software.

**Comment by deepak1556** [comment]
There is data in the runtime on the current double click interval configured in the OS https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:ui/views/metrics_aura.cc;l=14-22 which is wired to native text inputs, we could maybe expose that or just get the value from a native module.

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
